test/TestAssets/performance/XUnit1000Passing/XunitTests.tt

```diff
@@ -0,0 +1,49 @@
+﻿using Xunit;
+
+namespace XUnitAdapterPerfTests
+{
+	public class XuniTests
```

Comment:
There is a mix of spaces and tabs, could you save the file from VS to have it fixed please?